third_party/ObjexxFCL/src/ObjexxFCL/Array1S.hh

```diff
@@ -1111,7 +1111,7 @@ public: // Subscript
 	T const &
 	operator []( size_type const i ) const
 	{
-		assert( i < std::numeric_limits< int >::max() );
+		assert( i < std::numeric_limits< size_type >::max() );
```

Comment:
I don't know if anyone else was having a bunch of compiler warnings about this or just me.  `std::numeric_limits<int>::max()` was returning an integer which was being compared to a `size_type`.  I just changed it to look for the max value on size_type.